src/libNode/Node.cpp

```diff
@@ -1136,6 +1136,14 @@ bool Node::ProcessSubmitMissingTxn(const bytes& message, unsigned int offset,
     return true;
   }
 
+  if (offset >= message.size()) {
```

Comment:
should this be `>` or `>=` ?

Reply:
Originally i using >=, but it will break a lot of unit test case in Account

Reply:
Now changed to >=, please review again @bb111189